File: src/main/java/com/heybys/oddments/fooddelivery/persistence/cart/CartJpaRepository.java
package com.heybys.oddments.fooddelivery.persistence.cart;

import org.springframework.data.jpa.repository.JpaRepository;

import com.heybys.oddments.fooddelivery.domain.cart.Cart;
import com.heybys.oddments.fooddelivery.domain.cart.CartId;

interface CartJpaRepository extends JpaRepository<Cart, CartId> {}

File: src/main/kotlin/com/heybys/oddments/fooddelivery/persistence/cart/CartJpaRepository.kt
package com.heybys.oddments.fooddelivery.persistence.cart

import com.heybys.oddments.fooddelivery.domain.cart.Cart
import com.heybys.oddments.fooddelivery.domain.cart.CartId
import org.springframework.data.jpa.repository.JpaRepository

internal interface CartJpaRepository : JpaRepository<Cart, CartId>
